llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp

```diff
@@ -97,6 +100,63 @@ MDNode *buildSpirvDecorMetadata(LLVMContext &Ctx, uint32_t OpCode,
   return MDNode::get(Ctx, MD);
 }
 
+/// Builds a metadata node for a SPIR-V decoration for cache controls
+/// where decoration code and value are both uint32_t integers.
+/// The value encodes a cache level and a cache control type.
+///
+/// @param Ctx    [in] the LLVM Context.
```

Comment:
The comment does not seem to match the signature. Can you pls check? Thanks

Reply:
Fixed.